library/ix-dev/charts/photoprism/to_keep_versions.md

```diff
@@ -0,0 +1,4 @@
+# 1.1.13
+
+This version is kept because it contains a fix that is needed for migration to v2.x.x
+It should be safe to remove few months after v2.x.x is released.
```

Comment:
Might want to give a specific date/month to remove the version. IMHO something conservative like 6months from today should be more than sufficient.